.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "doubtfire-web"]
 	path = doubtfire-web
-	url = https://github.com/doubtfire-lms/doubtfire-web.git
+	url = git@github.com:thoth-tech/doubtfire-web.git
 [submodule "doubtfire-api"]
 	path = doubtfire-api
-	url = https://github.com/doubtfire-lms/doubtfire-api.git
+	url = git@github.com:thoth-tech/doubtfire-api.git
```

Comment:
**suggestion:** Similar to the suggestion above, could we keep this as HTTPS so it is consistent with the `doubtfire-overseer` repo below?